template/lambda-sqs-worker-cdk/README.md

```diff
@@ -14,9 +14,10 @@ Next steps:
 3. [ ] Add the repository to BuildAgency;
        see our internal [Buildkite Docs] for more information.
 4. [ ] Add Datadog extension, deployment bucket configuration and data classification tags to [infra/config.ts](infra/config.ts).
-5. [ ] Push local commits to the upstream GitHub branch.
-6. [ ] Configure [GitHub repository settings].
-7. [ ] Delete this checklist 😌.
+5. [ ] Configure smoke test `MessageAttributes` in [src/services/pipelineEventSender.ts](src/services/pipelineEventSender.ts) and `sourceSnsTopicArn` in [infra/config.ts](infra/config.ts).
```

Comment:
>Configure smoke test `MessageAttributes` in [src/services/pipelineEventSender.ts](src/services/pipelineEventSender.ts)

Is this needed?

Reply:
updated

additional permission to allow lambda to publish sns may be needed, I added in readme

Reply:
Do you think we should configure the permission by default?